substrate/frame/broker/src/nonfungible_impl.rs

```diff
@@ -50,3 +50,7 @@ impl<T: Config> Transfer<T::AccountId> for Pallet<T> {
 		Self::do_transfer((*index).into(), None, dest.clone()).map_err(Into::into)
 	}
 }
+
+// We don't allow any of the mutate operations, so the default implementation is used, which will
+// return `TokenError::Unsupported` in case any of the operations is called.
+impl<T: Config> Mutate<T::AccountId> for Pallet<T> {}
```

Comment:
Ideally there would be a simple test for this in case the defaults are changed

Reply:
Added a test to cover this.